projects/Website Blocker/website_unblocker.py

```diff
@@ -0,0 +1,18 @@
+import platform
+
+if platform.system() == "Windows":
+        pathToHosts=r"C:\Windows\System32\drivers\etc\hosts"
+elif platform.system() == "Linux":
+        pathToHosts=r"~\etc\hosts"
+
+pathToHosts=r"C:\Windows\System32\drivers\etc\hosts"
```

Comment:
**Remove this**, what happens write now is if the platform is `linux`, `pathToHosts` will be the windows directory